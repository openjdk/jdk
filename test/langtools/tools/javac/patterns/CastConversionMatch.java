/*
 * @test /nodynamiccopyright/
 * @bug 8231827
 * @summary Match which involves a cast conversion
 * @compile/fail/ref=CastConversionMatch.out -XDrawDiagnostics CastConversionMatch.java
 * @compile --enable-preview --source ${jdk.version} CastConversionMatch.java */

public class CastConversionMatch {
    public static void meth() {
        Object o = 42;
        if (o instanceof int s) {
            System.out.println("Okay");
        } else {
            throw new AssertionError("broken");
        }
        System.out.println(">Test complete");
    }
}
