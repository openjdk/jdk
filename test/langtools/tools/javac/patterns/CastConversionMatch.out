CastConversionMatch.java:11:26: compiler.err.preview.feature.disabled.plural: (compiler.misc.feature.primitive.patterns)
1 error
